bin/oneoff/move_user_storage_ids_to_dashboard.rb

```diff
@@ -0,0 +1,69 @@
+require File.expand_path('../../../pegasus/src/env', __FILE__)
+require src_dir 'database'
+require 'optparse'
+
+# To run: ruby bin/oneoff/move_user_storage_ids_to_dashboard.rb
+# To revert: ruby bin/oneoff/move_user_storage_ids_to_dashboard.rb --revert
+
+# Parse options
+options = {
+  revert: false,
+}
+
+OptionParser.new do |opts|
+  opts.banner = <<~BANNER
+    Usage: #{File.basename(__FILE__)} [options]
+
+    This moves the user_storage_ids table to the Dashboard DB and renames it to user_project_storage_ids
+
+    Options:
+  BANNER
+
+  opts.on('--revert',
+    'Will revert the Database to its previous state (drop the view and move the table back to Pegasus)'
+  ) do |revert|
+    options[:revert] = revert
+  end
+
+  opts.on('-h', '--help', 'Prints this help message') do
+    puts opts
+    exit
+  end
+end.parse!
+puts "Options: #{options}"
+options.freeze
+
+$revert = options[:revert]
```

Comment:
Mostly curious, was there a reason why we needed this variable (instead of just referencing it once directly below)?  (No need to change.)

Reply:
I was using variables in my last migration.. and I can't remember why but I figured there was a reason, so I kept it. I'll have to look into that.

Reply:
Just curious, no need to spend more time on it.  Thanks!